### PR TITLE
Feature/json server

### PR DESCRIPTION
**Pull request by kwbak**

### CODE REVIEW

File: api/data.json [hunk 1]
```diff
@@ -0,0 +1,643 @@
+{
```
Comment (mgolawski) [inline api/data.json]: Why do we have 4 spaces indenting here, while whole codebase has 2 spaces?

File: api/data.json [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+                "status": "SOURCED"
+            },
+            "countryOfIncorporationCode": {
+                "source": "KOMPANY",
+                "value": "",
+                "status": "SOURCED"
+            }
+        }
+    }
+}
```
Comment (mgolawski) [inline api/data.json]: Missing new line at the end of file.

File: api/data.json (hunk 1 above)
Comment (mgolawski) [reply api/data.json]: What about grid list mock?

File: api/data.json (hunk 2 above)
Comment (kwbak) [reply api/data.json]: I have editorconfig file, I use formatting tool, I don't know why there is no new line? I'll fix that.

> I think that we need some instruction inside README, how to install and use that mocks in the development mode.

Package json-server is dev-dependencies, I add script to npm `npm run json-server` - should I add sth about that to README?

**Pull request merged**